node/node.go

```diff
@@ -113,8 +113,8 @@ func New(conf *Config) (*Node, error) {
 		eventmux:      new(event.TypeMux),
 		log:           conf.Logger,
 		stop:          make(chan struct{}),
-		server:        &p2p.Server{Config: conf.P2P},
-		databases:     make(map[*closeTrackingDB]struct{}),
+		// server:        &p2p.Server{Config: conf.P2P},
```

Comment:
Should we just remove this? or should we keep it for posterity? (same below)

Reply:
I'm in favor of commenting out as much of the stuff we aren't actually using as possible. I think for a future update, having the code commented out instead of deleted will make upgrading to a newer version of geth much easier.